sdk/js-cloud-server/src/cloudClient.ts

```diff
@@ -152,20 +167,23 @@ export class DevCycleCloudClient {
                         (err as any).message
                     }`,
                 )
-                return {}
+                return {} as DVCVariableSet
             })
     }
 
-    allFeatures(user: DevCycleUser): Promise<DVCFeatureSet> {
+    async allFeatures(user: DevCycleUser): Promise<DVCFeatureSet> {
```

Comment:
these methods were already returning promises, and it doesn't seem like you're using await in them